launch.py

```diff
@@ -11,12 +11,12 @@
 
 python = sys.executable
 git = os.environ.get('GIT', "git")
-torch_command = os.environ.get('TORCH_COMMAND', "pip install torch==1.12.1+cu113 torchvision==0.13.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113")
+torch_command = os.environ.get('TORCH_COMMAND', "pip install torch torchvision --extra-index-url https://download.pytorch.org/whl/cu116")

```

Comment:
Should these torch packages be pinned to a specific version that knowingly works with cu116, just like how the previous packages were? This might break in the future

Reply:
this appears to be the same command given by pytorch website itself, so its probably good https://pytorch.org/get-started/locally/
![chrome_kOwmRPGPzj](https://user-images.githubusercontent.com/6380270/194871851-242986fa-24b4-41a4-a326-8b687a28b55f.png)